lib/pure/collections/tables.nim

```diff
@@ -2342,6 +2342,16 @@ proc inc*[A](t: var CountTable[A], key: A, val = 1) =
     if val != 0:
       insertImpl()
 
+proc dec*[A](t: var CountTable[A], key: A, val = 1) =
+  ## Decrements `t[key]` by `val` (default: 1).
+  runnableExamples:
+    var a = toCountTable("aaabbbbbbbbbbb")
+    a.dec('a')
+    a.dec('b', 10)
+    doAssert a == toCountTable("aab")
```

Comment:
assert is preferred in runnableExample on condition that the code is not compiled with the `danger` option.

ref https://nim-lang.github.io/Nim/contributing.html#best-practices